.github/workflows/validate.yml

```diff
@@ -197,13 +140,24 @@ jobs:
         laravel-version:
           - ^8
 
+    services:
+      mysql:
+        image: mysql:5.7
```

Comment:
We should consider at some point upgrading the mysql version (separate task ofc.)

Reply:
It is still the most popular version though: https://www.eversql.com/mysql-8-adoption-usage-rate/

Reply:
Also, see https://github.com/nuwave/lighthouse/issues/1784.